src/datasources/prices-api/coingecko-api.service.ts

```diff
@@ -17,6 +23,11 @@ export class CoingeckoApi implements IPricesApi {
    */
   private static readonly pricesProviderHeader: string = 'x-cg-pro-api-key';
 
+  /**
+   * Coingecko API maximum amount of token addresses being requested in the same call.
+   */
+  private static readonly maxBatchSize: number = 100;
```

Comment:
For batched requests to the Coingecko API, this `maxBatchSize` value is provided to avoid `414 - URI too large` style errors. (I've tested that kind of error happens with 150+ addresses, but this is set to `100` to stay conservative).